### PR TITLE
Update navbar, moved all pages in to home, added sections

### PR DESCRIPTION
**Pull request by waidelacruz**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -3,16 +3,16 @@
...
-  <link rel="stylesheet" href="{{site.baseurl}}/assets/site.css">
-  <link rel="alternate" type="application/rss+xml" title="SMC Style Guide" href="/feed.xml">
-  
-  
+	<link rel="stylesheet" href="{{site.baseurl}}/assets/main.css">
+	<link rel="alternate" type="application/rss+xml" title="SMC Style Guide" href="/feed.xml">
+
+	<script src="https://use.typekit.net/xzs4qjr.js"></script>
+	<script>try{Typekit.load({ async: true });}catch(e){}</script>
 </head>
```
Comment (tatwell) [inline _includes/head.html]: Make formatting generally and indentation generally part of the style guide.

What is our rule, team, for HTML? 2 or 4 spaces?

Personally, I favor 2. But I don't care so long as it is consistent.

This looks like 6-space indentation, which I am definitely against.

@waidelacruz 
@mandythomas 
@mirabelrice 
@kaylaziegler

Comment (tatwell) [reply _includes/head.html]: Once we establish a rule,  please update your text editor to enforce. And do call out inconsistencies in any PRs.

File: assets/colors.scss [hunk 2]
```diff
@@ -1,28 +1,31 @@
...
+$copa-dance: 		#007165;
+$copa-music: 		#793141;
+$copa-theater: 		#002e5f;
+$crean:		 		#1f8a84;
+$dodge:  			#2f2f2f;
+$fowler:  			#002f7c;
+$schmid:			#4a5E0f;
+$communication:		#c88508;
+$pharmacy: 			#959300;
+$wilkinson:			#008ec1;
```
Comment (tatwell) [inline assets/colors.scss]: Inconsistent formatting. Please fix.

Comment (waidelacruz) [reply assets/colors.scss]: This is fixed.

File: _includes/head.html (hunk 1 above)
Comment (waidelacruz) [reply _includes/head.html]: Switched to 4 spaces.

Comment (tatwell) [reply _includes/head.html]: This still looks like 6 spaces to me.

File: assets/colors.scss (hunk 2 above)
Comment (tatwell) [reply assets/colors.scss]: These are still not lined up.

File: assets/site.scss [hunk 3]
```diff
@@ -3,261 +3,74 @@
...
-  box-shadow:0px 1px 0px 0px rgba(0,0,0,0.1),0px 2px 1px 0px rgba(0,0,0,0.12);
+	margin-top: 0.5em;
+	border: solid $black 0.8px;
+	border-radius: 3px;
+	display: inline-block;
+	width: 220px;
+	box-sizing: border-box;
+	text-align: center;
+	padding: 15px;
+	box-shadow:0px 1px 0px 0px rgba(0,0,0,0.1),0px 2px 1px 0px rgba(0,0,0,0.12);
```
Comment (tatwell) [inline assets/site.scss]: These settings to look 6-spaces.

Team, what rule do we want for CSS file, 2 or 4 spaces?

**Review by tatwell — COMMENTED** [review]
@waidelacruz, review the Filed Changed tab in PR on Github and you'll see spacing is not yet fixed.

As a rule, after issuing a PR, you should review it yourself.

**Comment by khafa100** [comment]
@waidelacruz Outside of indentation everything looks fine to me. One thing I noticed is that the `_pages` folder isn't being used despite having files inside. Since you went with sections over pages for the site, we can remove that folder and replace it with a `_sections` folder. Then we can move the code in `_layouts/home.html` into `_sections` so that we don't have one big file and it'll be easier to keep things organized and find each section in the future. We have to configure jekyll for that, but I know that it can be done.

@tatwell If you want, I can take care of that for Wai in a future story. Or we can leave it as is for now. Up to you.

**Review by tatwell — APPROVED** [review]
👍

File: _pages/typography.md [hunk 4]
```diff
@@ -4,37 +4,93 @@ title: Typography
...
+				font-weight: 400;<br />
+				font-style: italic;
+				</span>
+			</div>
+			<div>
+				<p>Maecenas sed aliquet nulla. Cras sagittis posuere vestibulum. Nullam urna erat, malesuada eget justo at, <a href="#">fermentum cursus nibh</a>. Quisque non erat sagittis, pretium felis id, eleifend nisi. <a href="#">Vivamus egestas arcu</a> at posuere faucibus. Vestibulum interdum ipsum eu risus maximus convallis. Etiam viverra mattis orci, eget iaculis ipsum blandit ac.</p>
+				<p><a href="#">Fusce scelerisque semper nisl »</a></p>
+			</div>	
+		</div>
+	</div>
```
Comment (tatwell) [inline _pages/typography.md]: This file still looks like 6-space indenting.

File: assets/site-bak.scss [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+ .clear {
+		clear:both;
+}
+
+.new_icons {
+	display: inline-block;
+	width: 15%;
+	height: 80px;
+	padding-bottom: 50px;
+}
```
Comment (tatwell) [inline assets/site-bak.scss]: This file, too, still looks like it's 6-spaces.

**Comment by kaydanzie** [comment]
I was told 2 spaces when I did 4 a while ago, so I thought we were doing 2.

File: assets/site-bak.scss (hunk 5 above)
Comment (waidelacruz) [reply assets/site-bak.scss]: Oops this file is meant to be deleted.

File: _pages/typography.md (hunk 4 above)
Comment (waidelacruz) [reply _pages/typography.md]: This file too is meant to be deleted.

**Comment by kaydanzie** [comment]
Two space indents: https://kb-smc.chapman.edu/?p=1982

**Review by tatwell — APPROVED** [review]

**Pull request merged**